examples/windowing/noWindowExample/src/ofApp.h

```diff
@@ -0,0 +1,27 @@
+#pragma once
+
+#include "ofMain.h"
+
+class ofApp : public ofBaseApp{
+	
+	public:
+		
+		void setup();
+		void update();
+		void draw();
```

Comment:
Get rid of everything after `void update()` since it won't have any effect.  Not having it there underlines that fact (and also reminds users that ofBaseApp doesn't require overrides :)